Plugins.cpp

```diff
@@ -52,8 +52,20 @@ PluginInfo::PluginInfo() {
 }
 
 Plugins::Plugins(QObject *p) : QObject(p) {
-	qsSystemPlugins = QLatin1String("plugins/");
-	qsUserPlugins = QLatin1String("./");
+	// Current directory plugins
+	qsCurrentDirectoryPlugins = QDir::currentPath();
+	// System plugins directory
+	QDir qdSystemPluginsDir_x64("C:/Program Files/Mumble/Plugins");
+	QDir qdSystemPluginsDir_x86("C:/Program Files (x86)/Mumble/Plugins");
```

Comment:
Actually, for 1.3.x, you can't find the plugins this way.

Plugins live in a versioned directory. For example:

`C:\Program Files\Mumble\Versions\1.3.0~969~gbc12e8e~snapshot\plugins`